### ISSUE TITLE
Decreased androstenedione serum concentration

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6884]
**Preferred term label:**
Decreased androstenedione serum concentration

**Definition (free text, please give PubMed ID)**
Decrease of androstenedione level  in serum 
20301592[PMID]; 3312397[PMID]

**Parent term (use hpo.jax.org/app)**
Abnormal circulating androgen level (HP:0030347)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Congenital adrenal hyperplasia due to cytochrome P450 oxidoreductase deficiency (ORPHA 95699)

**Comment by pnrobinson** [comment]
added as HP_0033812
